guide/glossary/node.md

```diff
@@ -0,0 +1,219 @@
+   mobile = "/assets/images/guide/glossary/node/node-mobile.jpg"
+   mobileRetina = "/assets/images/guide/glossary/node/node-mobile@2x.jpg"
+   alt-text = "Network graph of Bitcoin node connections"
+   width = 1600
+   height = 500
+   layout = "full-width"
+%}
+
+# Node
+
+A node refers to a Bitcoin client that participates in the Bitcoin network. Any Bitcoin application that wants to show and send transactions needs a way to connect to a node and manage block data and there are many ways to make this work.
+
+There are multiple aspects that differentiate nodes:
+- Is it controlled by you or someone else? How much trust is involved?
+- Does it verify the data it receives?
+- Does it store all transaction data or only data relevant to you?
+- Security considerations, such as the risk of receiving manipulated data
+- Privacy considerations, such as tracing your transaction behavior
+
+## Common node setups
+
+From complete control to fully trusting a single third party, there are a variety of node setups. Below are examples of common ones.
+
+#### Local node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/local-node.png"
+   retina = "/assets/images/guide/glossary/node/local-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+A very simple setup is to use an application like Bitcoin Core that combindes node and wallet functionality.
+
+</div>
+
+#### Hardware node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/hardware-node.png"
+   retina = "/assets/images/guide/glossary/node/hardware-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+Separating node and wallet functionality, many users rely on a dedicated computer like a Rasperry Pi running software like Umbrel, MyNode, and Raspiblitz. Wallet applications can connect to the node which allows them to be more lightweight.
+
+</div>
+
+#### Self-hosted node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/own-hosted-node.png"
+   retina = "/assets/images/guide/glossary/node/own-hosted-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+While it possible to run a node on a rented server, this partially places the node in the control of a third party and introduces privacy risks.
+
+</div>
+
+#### Third-party node
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/glossary/node/third-party-node.png"
+   retina = "/assets/images/guide/glossary/node/third-party-node@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+A convenient setup is to connect to a node controlled by a third-party, which is only recommended if there is a good trust relationship.
```

Comment:
What is the extent of control mean? Who owns this third party node? Can I own it, or partially own it?

Reply:
It's meant as the node being out of your hands altogether - you have no control over it and may not even know who owns it.

Reply:
Ok and the reason you would be using such node is so that you can broadcast your transactions, and retrieve block data.

Otherwise there are no contributions to the network as you are not validating transactions or mining. You are trusting that the node operator gives you the correct data. As this can be a point of centralisation if too many people rely on such node operators it is usually discouraged.

Reply:
Exactly. It's maybe fine if a family member is controlling the node, or if you're in some other formal trust relationship like the community-managed setup used by Galoy. Good for starters or when the economic cost of running a node is too high and there's good trust relationship. It has its place, but should be considered carefully.

Reply:
> This setup is good for starters or when the economic cost of running a node is too high and there's good trust relationship. It is primarily used to broadcast transactions, and retrieve block data otherwise there are no contributions to the network as you are not validating transactions or mining. You are trusting that the node operator gives you the correct data, you have no control over the node and may not even know who operates it. As this can be a point of centralisation if too many people rely on such node operators it is usually discouraged.

Reply:
Great. Wanna post it as a suggestion and I'll merge?

Reply:
Well that could probably could use some more word smithing. If you think there's anything useful in there just copy and adapt.

Reply:
Just rewrote the text with the points you made.